lib/octokit/client/source_import.rb

```diff
@@ -9,9 +9,9 @@ module SourceImport
       # Start a source import to a GitHub repository using GitHub Importer.
       #
       # @param repo [Integer, String, Hash, Repository] A GitHub repository.
-      # @param vcs [String] The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
       # @param vcs_url [String] The URL of the originating repository.
       # @param options [Hash]
+      # @option options [String] :vcs The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
```

Comment:
The Source Imports API no longer requires the `vcs` attribute to start an import. I wanted to bring particular attention to this change as it breaks the current implementation by moving :vcs to the options hash.